fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -68,7 +68,8 @@ struct TenantManagementWorkload : TestWorkload {
 	int64_t maxId = -1;
 
 	const Key keyName = "key"_sr;
-	const Key testParametersKey = "test_parameters"_sr;
+	const Key testParametersKey =
+	    "test_parameters"_sr; // nonMetadataSystemKeys.begin.withSuffix("/tenant_test/test_parameters"_sr);
```

Comment:
```suggestion
	const Key testParametersKey = "test_parameters"_sr;
```

Reply:
We'd better put this under system key space, because it will cause `illegal_tenant_access`

Reply:
Currently the code here hasn't changed, but just a comment has been added. Are you saying that this doesn't work as written and needs to specify a key in the system key-space? That makes sense to me if it doesn't work in regular keys.

Reply:
> this doesn't work as written and needs to specify a key in the system key-space?

Yes, because we enabled the check we should put the test_parameters under system space